lightkurve/targetpixelfile.py

```diff
@@ -563,6 +564,8 @@ def plot(self, ax=None, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
             Whether or not to show the colorbar
         mask_color : str
             Color to show the aperture mask
+        context : str
+            matplotlib.pyplot.style.context, default is ggplot
```

Comment:
default looks like to be `'fast'`?

Reply:
Although the previous default was to do a scatter plot. I think I prefer it as a line plot...

Reply:
what I mean is that the docstring should say "`default is 'fast'`"